app/static/js/map.js

```diff
@@ -263,25 +271,40 @@ const displayResult = function (map, pixel) {
     }
 };
 
-export function initializeMaps() {
+const $mapContainer = $(".map__container");
```

Comment:
Add the titles / names of datasets to the screenreader portion of each map when the dataset shapes are added to the map